astroid/nodes/scoped_nodes.py

```diff
@@ -524,6 +523,8 @@ def __init__(
         """
         self.future_imports = set()
 
+        super().__init__(lineno=1, parent=parent)
```

Comment:
```suggestion
        super().__init__(lineno=0, parent=parent)
```
Seems like the `lineno` needs to be `0`.